pkgs/development/python-modules/flake8-polyfill/default.nix

```
@@ -11,6 +11,16 @@ buildPythonPackage rec {
     sha256 = "1nlf1mkqw856vi6782qcglqhaacb23khk9wkcgn55npnjxshhjz4";
   };
 
+  patches = [
+    # Notified upstream. Check if this is merged before the next release.
+    # TODO: Update this patch when there is a version upgrade to flake8.
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/PyCQA/flake8-polyfill/pull/14.patch";
```

technically not a stable url, so contents are free to change
```suggestion
      url = "https://github.com/PyCQA/flake8-polyfill/pull/14/commits/c531c24302f1414904b7e0af352e1ee7b91f1e09.patch";
```